### DISCUSSION TITLE
Install pybricks firmware using Chromebook?

### DISCUSSION THREAD

**Discussion by JamesP6000** [pybricks/support#1826] [Q&A] [answered]
Hi, Is it possible to install the pybricks firmware on a Spike Prime hub using a chromebook? As the instructions indicate, I first press the Bluetooth button until it starts blinking various colors. Then I click 'next' and a pop-up window appears but the hub never shows up...

To be clear, I'm connecting the hub to the chromebook via USB cable for this procedure.

**Comment by laurensvalk** [comment]
Yes, Chromebook is supported.

Perhaps double check the USB cable or try another? If it's a charging-only cable, the lights will turn on but it can't transfer any data. (This is a common question - apologies if you've already tried this)

**Comment by JamesP6000** [comment, marked as answer]
I had tried two USB cables but it ends up that both of them had issues :( The third cable worked just fine! Sorry for the bother!

**Reply by laurensvalk** [reply]
No worries. Glad it worked!
